webserver/webserver/src/test/resources/logging-test.properties

```diff
@@ -22,7 +22,9 @@ java.util.logging.SimpleFormatter.format=%1$tY.%1$tm.%1$td %1$tH:%1$tM:%1$tS %4$
 #All log level details
 .level=INFO
 
-io.helidon.webserver.level=INFO
-io.helidon.common.reactive.level=INFO
+io.helidon.webserver.level=FINE
+#io.helidon.webserver.HttpInitializer.level=FINE
+io.helidon.common.reactive.level=ALL
+io.netty.handler.logging.LoggingHandler.level=ALL
```

Comment:
Please revert log levels to original, we should not have full logs during testing.